frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -186,27 +186,29 @@ const ClusterVersionDetailsTable: React.SFC<ClusterVersionDetailsTableProps> = (
     </div>
     <div className="co-m-pane__body">
       <SectionHeading text="Update History" />
-      <div className="co-table-container">
-        <table className="table">
-          <thead>
-            <tr>
-              <th>Version</th>
-              <th>State</th>
-              <th>Started</th>
-              <th>Completed</th>
-            </tr>
-          </thead>
-          <tbody>
-            {_.isEmpty(history) && <EmptyBox label="History" />}
-            {history.map((update, i) => <tr key={i}>
-              <td className="co-break-all">{update.version || '-'}</td>
-              <td>{update.state || '-'}</td>
-              <td><Timestamp timestamp={update.startedTime} /></td>
-              <td>{update.completionTime ? <Timestamp timestamp={update.completionTime} /> : '-'}</td>
-            </tr>)}
-          </tbody>
-        </table>
-      </div>
+      {_.isEmpty(history)
```

Comment:
We might need some design input on this since we are changing the way the empty state is displayed. In most other places we replace just the table content with empty state, but this will replace the entire table. Maybe use `<td colspan="4">` and put the `EmptyBox` inside that?

Reply:
> We might need some design input on this since we are changing the way the empty state is displayed.
I dont think we are.

Eg. Empty ConfigMaps list:
![2](https://user-images.githubusercontent.com/1668218/52668989-5053d200-2f15-11e9-89c5-da8f148cb79b.png)

Empty History list:
![1](https://user-images.githubusercontent.com/1668218/52668998-5649b300-2f15-11e9-9c66-b4e9126ac91c.png)


Also if you check other places we are using EmptyBox component its pretty much the same:
- https://github.com/openshift/console/blob/master/frontend/public/components/factory/list.tsx#L332
- https://github.com/openshift/console/blob/master/frontend/public/components/cluster-settings/oauth.tsx#L27

Reply:
Thanks for following up. Fair point that there are exceptions. I also took a look at how 'Conditions' and container tables are handled since they have a similar context, and they replace the entire table with an empty state (or with containers, no empty state at all). This is probably fine.